swift/ql/lib/codeql/swift/security/SensitiveExprs.qll

```diff
@@ -64,9 +64,9 @@ class SensitivePrivateInfo extends SensitiveDataType, TPrivateInfo {
   override string toString() { result = "private information" }
 
   override string getRegexp() {
+    // we've had good results for the e-mail heuristic in Swift, which isn't part of the default regex. Add it in.
     result = HeuristicNames::maybeSensitiveRegexp(SensitiveDataClassification::private())
-    or
-    result = "(?is).*e(mail|_mail).*"
+      .replaceAll(".*(", ".*(e(mail|_mail)|")
```

Comment:
In my local tests it didn't have a measurable effect either way, but those tests were small.  I've started a (second) DCA run to get a proper answer.

Past experience is that additional regular expressions for matching sensitive expressions are very expensive, because they're compared against a lot of strings from the database - whereas making one regex larger has a much smaller cost (presumably because the regex engine is able to compile it into something efficient).

I don't think the new string is a problem here because the regex string itself should only be assembled once (I hope).